src/hooks/useGridKeyboardNavigation/__tests__/useGridKeyboardNavigation.jest.js

```diff
@@ -0,0 +1,100 @@
+import { renderHook, cleanup, act } from "@testing-library/react-hooks";
+import { fireEvent } from "@testing-library/react";
+import range from "lodash/range";
+import useGridKeyboardNavigation from "../useGridKeyboardNavigation";
+import { NAV_DIRECTIONS } from "../../useFullKeyboardListeners";
+
+describe("useGridKeyboardNavigation", () => {
+  let element;
+
+  afterEach(() => {
+    element.remove();
+    cleanup();
+  });
+
+  function itemsArray(length) {
+    return range(length);
+  }
+
+  function renderHookForTest({ items = itemsArray(4), numberOfItemsInLine = 3, onItemClicked = jest.fn(), focusOnMount = false }) {
```

Comment:
just my opinion - maybe we can move this function and other helpers outside the description ? 
i find it easier to read the tests

but it is just my opinion

Reply:
I can do that, but then I'll need to move some stuff outside the `describe` block (e.g. the `element` var).
WDYT about moving the functions to the end of the `describe`? I think it will increase readability without affecting the scope

Reply:
sure, lets do it